files/minio.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Description=MinIO
+Documentation=https://docs.min.io
+Wants=network-online.target
+After=network-online.target
+AssertFileIsExecutable=/usr/bin/minio
+
+[Service]
+WorkingDirectory=/usr
```

Comment:
To check; I suspect this is where minio stores in-flight uploads, before atomically moving them to the actual bucket directory. If so, it'll have to be a scratch directory under `/opt/minio`

Reply:
no, WorkingDirectory is a systemd construct, it's just the context that Exec gets done from, and we're defining that as an explicit path anyways so it doesn't matter.